src/style.css

```diff
@@ -1 +1,102 @@
-/* add styles here */
+/* stylelint-disable font-family-name-quotes */
+@font-face {
+  font-family: 'Avenir';
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir';
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+body {
+  font-family: 'Avenir', sans-serif;
+  font-weight: 300;
+  margin-top: 20px;
+  height: 768px;
+}
+
+.icon {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px;
+  padding-left: 62px;
+  display: block;
+}
+
+.icon-2 {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 12px;
+  padding-left: 33px;
+  background-size: 1.1%;
+  display: block;
+}
+
+.search {
+  display: flex;
+  align-items: center;
+  color: #3d4e61;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  transition: box-shadow;
+  font-family: 'Avenir', sans-serif !important;
+  font-weight: normal;
+  box-shadow: 0 1px 8px #3d4e6133;
+  font-size: 16px;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.search-2 {
+  display: flex;
+  align-items: center;
+  color: #3d4e61;
+  margin-top: 20px;
+  height: 42px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  font-family: 'Avenir', sans-serif !important;
+  font-weight: normal;
+  transition: box-shadow;
+  box-shadow: 0 1px 8px #3d4e6133;
+  text-shadow: 0 4px 3px #3d4e6133;
+  font-size: 14px;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px #3d4e6133;
+}
+
+.search:focus {
+  font-family: 'Avenir', sans-serif !important;
+  font-weight: bold;
+  box-shadow: 0 3px 8px #3d4e6133;
+  text-shadow: 0 4px 3px #3d4e6133;
+  outline: 0;
+}
+
+.search-2:hover {
+  box-shadow: 0 3px 8px #3d4e6133;
+}
+
+.search-2:focus {
+  font-family: 'Avenir', serif !important;
+  font-weight: bold;
+  box-shadow: 0 3px 8px #3d4e6133;
+  text-shadow: 0 0 0;
+  outline: 0;
+}
+
+.search::placeholder {
+  color: #3d4e61;
+}
+
+.search-2::placeholder {
+  color: #3d4e61;
+}
```

Comment:
you can create some class and all common styles there

it will simplify your code